spec/controllers/surveys_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'rails_helper'
 
 describe SurveysController, type: :controller do
+  context 'send survey validation' do
 
+    it 'respondents is empty' do
+      FactoryGirl.create(:survey, :id => 1)
```

Comment:
To reduce your typing, you can usually replace `:id => 1` syntax with `id: 1` in ruby!